cli/src/commands/rebase.rs

```diff
@@ -53,16 +53,35 @@ use crate::ui::Ui;
 ///
 /// There are three different ways of specifying which revisions to rebase:
 ///
-/// * `-b` to rebase a whole branch
-/// * `-s` to rebase a revision and its descendants
-/// * `-r` to rebase the specified revisions without their descendants
+/// * `--source/-s` to rebase a revision and its descendants
+/// * `--branch/-b` to rebase a whole branch, relative to the destination
+/// * `--revisions/-r` to rebase the specified revisions without their
+///   descendants
 ///
 /// If no option is specified, it defaults to `-b @`.
 ///
-/// With `-s`, the command rebases the specified revision and its descendants
-/// onto the destination. For example, `jj rebase -s M -d O` would transform
-/// your history like this (letters followed by an apostrophe are post-rebase
-/// versions):
+/// There are three different way of specifying where the revisions should be
+/// rebased to:
+///
+/// * `--destination/-d` to rebase the revisions onto of the specified targets
```

Comment:
I’m having a really hard time parsing the phrase “onto of the specified targets”. Should it be “onto the specified targets” or “onto one of the specified targets”? If so, which one if there are multiple? Or should it be “on top of the specified targets”?

Reply:
It should be the first one. Want to send a PR?

Reply:
Sure, I’ll give it a go tonight

Reply:
I think someone else actually already fixed it, but thanks anyway

Reply:
that was quick, thanks @cenviity!